java/ql/src/semmle/code/java/security/ResponseSplitting.qll

```diff
@@ -0,0 +1,11 @@
+import semmle.code.java.dataflow.DataFlow
```

Comment:
This file needs to start with a file-level qldoc immediately followed by `import java`.